docs/UserGuide.md

```diff
@@ -341,8 +337,18 @@ This will add an exam to a student by searching for their `ID`. Time is an optio
 **Format 1:** `addexam -id {ID} -exam {EXAMNAME} -date {DATE}` <br>
 **Format 2:** `addexam -id {ID} -exam {EXAMNAME} -date {DATE} -time {TIME}`
 
+**Notes on Fields for add exam command**
+
+| Field        | Prefix   | Required | Caveats                                                                                                                                                                     |
+|--------------|----------|----------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **uniqueId** | `-id`    | Yes      | Maximum 6 digits allowed but you can omit any leading 0s.<br> e.g. `1`, `01`, `000001` are all valid id inputs for the id `#000001`                                         |
```

Comment:
I'm concern about the phrasing of  "Maximum 6 digits allowed", cuz iD::isValid method also checks if the value is between 1 and 999999 (both inclusive). so 7 or more digits can be valid due to leading 0's

I scared for the PE, they might just test and say, but the below is valid so it's a bug there's a mismatch between UG and app.

ID 000000001 or
ID 00000009999 (if id 9999 exist)

Reply:
ok will change to positive integer with maximum 6 digits with leading 0s optional